src/createStreamMiddleware.ts

```diff
@@ -63,7 +63,7 @@ export default function createStreamMiddleware() {
   ) {
     let err;
     try {
-      const isNotification = !res.id;
+      const isNotification = !('id' in res);
```

Comment:
is this `true` if `id` is `null`?

Reply:
`isNotification` will be `false` if `res.id === null`.